mergebot_poller_test.py

```diff
@@ -153,7 +156,7 @@ def testSearchInvalidCommandPostFail(self, mock_merge_git, mock_comment,
                                     mock_pr):
         # Tests that an invalid comment on a valid PR is caught successfully,
         #  but if the post fails we still return false.
-        gp = mergebot_poller.GithubPoller({'repository': 'asdf'})
+        gp = mergebot_poller.GithubPoller(self.args)
```

Comment:
Why not do `gp = mergebot_poller.GithubPoller(self.args)` in the setup method. Then you don't need to store the args at all...